doc/source/user_guide/ug_running.rst

```diff
@@ -747,6 +747,15 @@ cases.  Current filenames can be found in the options scripts in
    about 4 :math:`W/m^2` for biases in the forcings or the model. In the namelist, set ``atm_data_type = clim`` 
    to use climatological atmospheric forcing.
 
+Horizontal ice advection
```

Comment:
I think that referring to any of these changes as "ice advection" is going to be confusing, since advection isn't technically done in the column physics, and there's no attempt at volume conservation, which _should_ always be conserved under advection.  Maybe the ``ice_advection_type`` option could be renamed something like ``surrounding_ice_type``?  And then make it explicit in the documentation that this allows the effects of advective processes to be represented in a single grid cell; in Icepack the uniform ice option is not conservative while the 'none' option is. ?

Reply:
I agree we don't want to confuse anyone. What if we renamed `ice_advection_type` to be `lateral_flux_type` and the choices could be either 'uniform_ice' or 'open_water' (with the default still being 'uniform_ice')? I think that makes it more clear that ice/open water is entering/exiting the grid cell. And, at least to me, 'flux' doesn't carry the same volume conservation connotation as 'advection'. Also, note that the 'none'/'open_water' option is only conservative when the forcing is net convergence. When the forcing is net divergence, we must remove ice volume from the single grid cell (i.e., the open water produced by a net divergence should reduce the mean thickness in the grid cell).

Below is a proposal for how to change the documentation to make this clear. Please let me know if this makes sense. Thanks!

Horizontal ice advection
------------------------

When Icepack is run in standalone (i.e., single column) mode with a dynamical forcing (e.g., ``ocn_data_type = SHEBA``),
closing implies the lateral flux of ice or open water area into the grid cell. The default assumption 
(in the namelist, ``lateral_flux_type = 'uniform_ice'``) is that the simulated grid cell is surrounded by grid cells with identical
ice properties to the simulated grid cell. When the dynamical forcing is net convergence, this uniform ice is fluxed into the
grid cell. Alternatively, one may assume that the simulated grid cell is surrounded by open water (in the namelist
``lateral_flux_type = 'open_water'``). Note that assuming the grid cell is surrounded by open water means that closing (i.ice convergence) will produce open water in the grid cell. In either case, when the forcing is net divergence, ice area
and volume are removed from the grid cell to accommodate the formation of open water implied by the net divergence.

Reply:
I like your suggestion to use `lateral_flux_type` instead.  Here's how I would do the documentation:

Horizontal ice advection
------------------------

When Icepack is run in standalone mode with a dynamical forcing (e.g., ``ocn_data_type = SHEBA``),
closing implies the lateral flux of ice or open water area into the grid cell. The default assumption 
(in the namelist, ``lateral_flux_type = 'uniform_ice'``) is that the active grid cell is surrounded by grid cells with
identical ice properties to the active grid cell, i.e. the ice is uniform across all of those cells, and when the dynamical forcing is net convergence, this uniform ice is 
fluxed into the grid cell. Alternatively, one may assume that the active grid cell is surrounded by open water (in the 
namelist ``lateral_flux_type = 'open_water'``), in which case 
closing (i.e., ice convergence) will produce open water in the grid cell. In either case, when the forcing is net 
divergence, ice area and volume are removed from the grid cell to accommodate the formation of open water implied 
by the net divergence.